packages/core/docs/guide/authentication.md

```diff
@@ -0,0 +1,74 @@
+# What's authentication?
+Authentication is the operation of recognizing the user's identify. It allows us to associate incoming requests with an account or a person. To do that, provided credentials are compared with ones in the database.
+
+The whole logic related to this process can be handled with methods shared by `useUser` [composable](https://v3.vuejs.org/guide/composition-api-introduction.html#basics-of-composition-api).
```

Comment:
```suggestion
The whole logic related to this process can be handled with methods shared by `useUser` composable
```

nice one!... but not in this case since we already have intro to composables in our own docs ;)